crates/package_manager/src/virtual_dir.rs

```diff
@@ -0,0 +1,79 @@
+use crate::{link_file, symlink_pkg, LinkFileError};
+use derive_more::{Display, Error};
+use miette::Diagnostic;
+use pacquet_lockfile::{
+    DependencyPath, PackageSnapshot, PackageSnapshotDependency, PkgNameVerPeer,
+};
+use pacquet_npmrc::PackageImportMethod;
+use rayon::prelude::*;
+use std::{
+    collections::HashMap,
+    ffi::OsString,
+    fs,
+    path::{Path, PathBuf},
+};
+
+#[derive(Debug, Display, Error, Diagnostic)]
+pub enum CreateVirtdirError {
```

Comment:
We don't lose anything by writing it in full, but lose the ability to read and understand quickly with using unpopular abbreviations

```suggestion
pub enum CreateVirtualDirectoryError {
```

Reply:
I agree but using Dir instead of Directory should be fine.

Reply:
I'm good with Dir instead of Directory 👍

Reply:
resolved by https://github.com/pnpm/pacquet/pull/143/commits/abfaddbc0c396dc35c2e2a9d359c2b671b88d02f